docs/source/notes/activelearning.rst

```diff
@@ -50,25 +53,70 @@ The active learning paradigm
+of training and evaluation.
+
+By nature, this is an iterative process following several *active learning rounds*
+starting from round :math:`k=0` to :math:`K>0` set by some labelling budget or we run out of observations to label.
+Given the training data at round :math:`k`, :math:`L_k`, The active learning practitioner's job is to design **strategies**
+that will select or suggest **informative** observations to an oracle capable of providing labels --- then, construct an improved
+training dataset :math:`L_{k+1}` which will improve the model's understanding of the underlying data distribution and hopefully
+improve it's performance on the task.
+
+The figure above summarises a generic active learning cycle. In this setup we have:
+
+* A dataset of observations :math:`D`, which is split into
+    * a labelled subset :math:`L`
+    * an unlabelled subset :math:`U`
+* A fixed model :math:`f_{\theta}`, which will be trained on :math:`L`
+* An oracle which can provide labels to unlabeled observations :math:`u \in U`
+* An active learning strategy, which in this case is split into
+    * a informativeness function :math:`g`
+    * a selection function SELECT, which selects observations based on the informativeness values coming from :math:`g`.
+
+Starting as before we use :math:`L` to train a model and produce :math:`f_{\hat{\theta}}`. We can optionally evaluate the
+performance of this model based on the current iteration of :math:`L` with a hold-out test set :math:`E`. In order to suggest
+useful observations to the oracle, we have to be able to compute the *informativeness* of observations.
+
+"Informativeness" is a measure which encapsulates how useful we believe labelling an observation will be to improving our model.
+Computing informativeness values stands at the very core of all active learning strategies. In Bayesian optimisation based active
+learning strategies, the acquisition function computes informativeness values so this term pops up often as well.
 Many different informativeness measures exist, one family of informativeness measures stems from the model's uncertainty about
 its prediction. The intuition is that observations with high uncertainty in the posterior predictive distribution are
-useful and hence "informative" [#f2]_. Applying this example to our figure, if :math:`g(\cdot)` a function for computing
-uncertainty over predictions, :math:`g(f_{\hat{\theta}}(u))` will give us an informativeness score for each :math:`u`
-that we can then rank and pick to be labelled by the oracle. Once labelled the cycle begins again, hopefully increasing
-our test performance in subsequent iterations.
+useful and hence "informative" [#f1]_.
+
+Our figure follows this logic. We use the trained model :math:`f_{\hat{\theta}}` to make predictions for each of the
+unlabelled observations :math:`f_{\hat{\theta}}(u) \textrm{for} u \in U` [#f2]_. :math:`g(\cdot)` is a function for computing
```

Comment:
We should align notations with the diagram wherever possible

Reply:
I agree! but I am not sure what is different here?